Obsidian.API/_Interfaces/ILiving.cs

```diff
@@ -16,4 +16,54 @@ public interface ILiving : IEntity
     public int AbsorbtionAmount { get; set; }
 
     public Vector BedBlockPosition { get; set; }
+    
+    /// <summary>
+    /// The list containing all currently active <see cref="PotionEffect"/>s.
+    /// </summary>
+    public IReadOnlyList<PotionEffect> ActivePotionEffects { get; }
```

Comment:
I feel like this should be a dictionary that holds more relevant information i.e a class that has duration, amplifiers, flags and ofc with the effect as its key ^ ^ instead of having what you did below with "PotionEffectDurations"

Reply:
@Tides changed it

Reply:
Looks good thanks!